### PR TITLE
More careful ordering for `@family` tag

### PR DESCRIPTION
**Pull request by krlmlr**
Needed for DBI: `dbSendQuery()` vs. `dbSendQueryArrow()` .

Test-first, the second commit afbabca036e000124711ae4cd538167113bcaa07 shows the effect. Happy to revisit the implementation and/or switch the other tests to snapshots.

I suspect that the new `order_c()` should be used almost everywhere where we now have `order()` .

### CODE REVIEW

File: tests/testthat/test-rd-family.R [hunk 1]
```diff
@@ -123,3 +123,29 @@ test_that("custom family prefixes can be set", {
 
   expect_match(out$get_value("seealso"), "^Custom prefix:")
 })
+
+test_that("careful ordering", {
+  # Can't use foo1 because it's used as class in another test
```
Comment (hadley) [inline tests/testthat/test-rd-family.R:128]: Is it an S4 class? It might need a `on.exit(removeClass("foo1"))` call.

**Review by hadley — APPROVED** [review]
Looks fine to me. I can get in the next roxygen2 release if you add a news bullet.

**Comment by krlmlr** [comment]
Thanks, good to go from my end.

File: tests/testthat/test-rd-family.R (hunk 1 above)
Comment (krlmlr) [reply tests/testthat/test-rd-family.R:128]: No, unfortunately, #1564 does not change the behavior here.

Comment (krlmlr) [reply tests/testthat/test-rd-family.R:128]: Happy to adapt if #1566 or a better version gets in.

**Comment by krlmlr** [comment]
Done, finally.

**Pull request merged**